### PR TITLE
relist StartPage in Search Engines

### PR DESCRIPTION
**Pull request by danarel**
Resolves: #1562 

This is still a topic in discussion, but this pull request would complete the relisting of StartPage should we choose to do so.

### CODE REVIEW

**Comment by danarel** [comment]
That sentence was copied from the previous description before de-listing.

**Comment by LizMcIntyre** [comment]
Dan is right that the wording "Behind StartPage is a European company that has been obsessive about privacy since 2006" is very old. I agree with Mikaela that it's "strange" since we know Startpage is now majority owned by U.S. company System1. 

For the sake of transparency, I believe the wording should be changed so consumers don't mistakenly believe Startpage is subject only to EU privacy laws. (It's a gray area, I'd say.) This matters A LOT to some people, and PTIO doesn't want to lose trust.

It's wise that you've gone with Dan Arel's earlier warning label recommendation for Startpage over ownership. I agree that consumers should be informed that Startpage (via the holding company) is now majority owned by System1 and that [System1 is involved in day-to-day processing of search data](https://support.startpage.com/index.php?/Knowledgebase/Article/View/1276/0/what-is-the-startpage-privacy-guarding-data-flow). (See the small print in that diagram that notes user personal information is fuzzed). 

I recommend sharing a link to [System1](https://www.system1.com/) instead of or in addition to the Startpage explanation now linked so consumers can evaluate System1 ownership themselves.The current link only refers to System1 as "a consumer internet company with a lot of search engine experience." Some consumers could feel misled when they find out that System1 is actually a pay-per-click behavioral advertising company.

I'm just seeing this github action now and apologize for the late input. It might be a good idea to post  proposed new listings or delistings in the regular PTIO community area and at r/privacytoolsio so more community participants can see them and provide input in a timely manner. I know some community members avoid using github now that it's owned by Microsoft.

**Comment by LizMcIntyre** [comment]
@Mikaela was that conflict of interest label for me? No problem, if so. I used to work with Startpage, but resigned Oct. 1. I have always made that very clear in my bios and if anyone asks me.

**Comment by JonahAragon** [comment]
This is still a complicated issue — I don't think we've reached any agreement. 

> was that conflict of interest label for me?

Just a misunderstanding :)

**Comment by Mikaela** [comment]
> @Mikaela was that conflict of interest label for me? No problem, if so. I used to work with Startpage, but resigned Oct. 1. I have always made that very clear in my bios and if anyone asks me.

No, we have a team member whom I cannot name and whom I hope will come up who has been offered a job by StartPage and we are having a bit of disagreement within the team on how to handle that.

In my opinion the person should tell about it by themselves as I fear it could sway people close to them in the team into accepting StartPage getting relisted in fear of making them lose the opportunity thus making this a conflict of interest and I see a potential conflict of interest even if the team swore to not care about their employment.

* * * * *

Hi @JonahAragon, nice to see you, could you look into these sometime? 

* https://github.com/privacytoolsIO/privacytools.io/pull/1125 - most important to my mental health and hopefully helping with what can become a burnout
* https://github.com/privacytoolsIO/privacytools.io/pull/1651 - proposed alternative to the _Conflict of Interest_ label (I do think we need both, better safe than sorry)
* https://github.com/privacytoolsIO/privacytools.io/issues?q=is%3Aopen+assignee%3AJonahAragon - multiple issues where no one else has access to
* add every team member as moderator at Discourse (no issue opened yet) - what I and at least @nitrohorse have been asking you from some time.
* possibly look into https://forum.privacytools.io/t/discussion-riot-im/665/18?u=mikaela as I don't have a Reddit account to personally verify that that person is a /r/privacy moderator, I would like to have that verified so their flair can be set as "/r/privacy moderator" like ours say "team" and they could possibly put a better "real name"

Good night, it's a bit over 01 am for me.

**Comment by blacklight447-ptio** [comment]
Anyway, i say we do not re add startpage, but we should keep an eye on them, they broke trust, causing them to be removed, quickly coming up with answers after that trust was broken doesn't make on immediately trusted again, the damage has already been done. However, being delisted does not mean that the trust cannot be restored, see matrix/riot as example, which got relisted after a significant amount of time by hard work.


We should keep an eye on startpage, for when it becomes better, or in another case, worse. If startpage can keep its independence for a long period without any actual changes, then i would be more willing to give in my vote, but for now, i vote to not re list them, and keep a close eye on the for any changes in the situation.

**Comment by LizMcIntyre** [comment]
Hi @blacklight447-ptio . **There may be another Startpage trust issue here if someone on the PTIO team was hired/contracted or offered the possibility of work by Startpage/System1**. 

I received a response to my post above regarding the conflict of interest label Mikaela posted, but that has been marked as off-topic so you have to sign in to see it. Some was off topic, but the part about a conflict of interest may not be. Here is that now hidden response: 

>Mikaela Jan 15, 2020 Member

>...we have a team member whom I cannot name and whom I hope will come up who has been offered a job by StartPage and we are having a bit of disagreement within the team on how to handle that.

>In my opinion the person should tell about it by themselves as I fear it could sway people close to them in the team into accepting StartPage getting relisted in fear of making them lose the opportunity thus making this a conflict of interest and I see a potential conflict of interest even if the team swore to not care about their employment.

I agree with @mikaela that any connection with Startpage/System1 or any related entities should be disclosed, including possible future connections that have been discussed or contemplated.

**Comment by LizMcIntyre** [comment]
Could someone on the PTIO team here detail the conflict of interest and how it arose? The person involved hasn't stepped up to explain. Thanks.

**Comment by blacklight447-ptio** [comment]
@LizMcIntyre hey there!

So the person who is offered a job prefers to not come out public with it just yet, as the its not even clear whether he/she is taking the job at all. As a privacy team we respect that wish. However i do understand the concern.

The is the first time one of us got offered a job like this will being part of ptio, so we are still discussing how we are dealing with it. In any case i can assure you that we won't let this affect the decision from ptio on this subject ( this is actually a good example of why all decisions are made in public, and no single person can add things to the site: you would have to corrupt the whole team, and also sneak it past the community.)


When the contract is signed or declined, im pretty sure the person would feel glad to explain the situation further.

**Comment by LizMcIntyre** [comment]
> @LizMcIntyre hey there!

Hi @blacklight447-ptio ! Thanks for the additional information.

> So the person who is offered a job prefers to not come out public with it just yet, as the its not even clear whether he/she is taking the job at all. As a privacy team we respect that wish. However i do understand the concern.

I understand. This is a *very* awkward situation.

Note that a conflict of interest (in fact or appearance) can exist even if the person isn't sure he or she will accept the offer. (Former auditor here!) 

> The is the first time one of us got offered a job like this will being part of ptio, so we are still discussing how we are dealing with it. In any case i can assure you that we won't let this affect the decision from ptio on this( or any other) subject ( this is actually a good example of why all editorial decisions are made in public, and no single person can add or remove things to the site: you would have to corrupt the whole team, and also sneak it past the community.)

I believe in the good intentions and value of the community, Blacklight. @Mikaela speaking out is evidence of this, and the Team should thank her for it.

> When the contract is signed or declined, im pretty sure the person would feel glad to explain the situation further.

Waiting to make disclosures could erode trust. There's already some buzz about this situation. If a conflict of interest exists, PTIO should be transparent about it.

The Startpage relisting thread probably isn't the best place for an extended discussion. Let's bring this situation to the PTIO Forum so the community can offer input.

**Comment by danarel** [comment]
Alright, I want to address the comments on this pull request.

I am going to give a lot of detail here in the hopes of clarifying this.

When the System1 investment into Startpage went down and the CEO contacted both Jonah and I to help answer the questions the privacy community had. Through those discussions and subsequent emails about how Startpage could have better handled the situation and why the privacy community was so alarmed, it was revealed that my professional background is in marketing and communications. Coupled with my experience and knowledge in the privacy community, I was offered a meeting w/ some of the Startpage team.

That meeting led to them offering me a contract to do 2 things.

1. To write a handful of blog posts for their blog related to their search engine, but also to privacy in general. This is something I already do professionally as a columnist, blogger, and author. Guest blogging is nothing new to me.

2. To meet with their team as a consultant and share my marketing/communications/privacy related experience with them.

As a professional marketer and writer, this is what I do. I will not be a Startpage employee or on their payroll.

That's it. Any compensation being given to me will be for these services, which are part of my professional expertise.

The moment I got off the call with Startpage, I alerted the PrivacyTools team about the potential offer and that I believed this could cause a conflict of interest and since this has not happened to any other member, I wanted to make them aware so we could decide how to best handle the potential conflict. Did that mean I would have to leave the team? I was not sure, but I was willing to do so if asked. The integrity of the site is important to me, regardless of my status as a team member. While we are still discussing it now, we all agree some guidelines should be put in place. I asked that the team not go public until we had internal discussions and that I was sure I was even going to accept or decline the offer.

When it comes to this pull request to relist Startpage, it should be noted that:

1. It is a PR in response to an issue opened by another team member who agreed that Startpage should be relisted based on the answers we got from those questions. The PR cannot be pushed live by me without multiple team member approval. This ensures that even if I had not notified the team of the pending contract, that I could not just re-list a service on my own. Not only would I have to convince them it was the right thing to do, but also the community. This is one of the great features of PrivacyTools.

2. The issue and PR predates the meeting I had with Startpage and I only created the PR to satisfy the issue, as you have seen done many times before on our Github.

Startpage has not asked me to relist their service even though I am sure they would love to be. What service wouldn't want to be? It's a fantastic resource privacy tools and is well respected by users, organizations, and companies.

I hope this helps clarify things.

**Comment by LizMcIntyre** [comment]
Thank you for clarifying @danarel. I agree with you that guidelines should be put in place to ensure PTIO integrity. I recommend taking this discussion to the Forum for community input.

**Comment by danarel** [comment]
Made an update to today to match the removal of JiveSearch to keep my fork up to date.

File: _includes/sections/search-engines.html [hunk 1]
```diff
@@ -19,19 +19,29 @@ <h1 id="search" class="anchor"><a href="#search"><i class="fas fa-link anchor-ic
...
 github="https://github.com/duckduckgo"
 %}
 
+{% include cardv2.html
+title="StartPage"
+image="/assets/img/svg/3rd-party/startpage.svg"
+description='StartPage is a search engine that provides Google search results with complete privacy protection. <span class="flag-icon flag-icon-nl"></span> Behind StartPage is a European company based in the Netherlands that has been obsessive about privacy since 2006.'
+labels="warning:<a href=//support.startpage.com/index.php?/Knowledgebase/Article/View/1277/0/startpage-ceo-robert-beens-discusses-the-investment-from-privacy-one--system1>Warning</a>:StartPage was recently acquired by an advertising company."
+website="https://www.startpage.com/"
+privacy-policy="https://www.startpage.com/en/privacy-policy/"
```
Comment (nitrohorse) [inline _includes/sections/search-engines.html:36]: We’ll want to link to privacy-policy=“https://www.startpage.com/en/privacy-policy/“

Comment (nitrohorse) [reply _includes/sections/search-engines.html:36]: Oh nvm :)

File: _includes/sections/search-engines.html [hunk 2]
```diff
@@ -19,19 +19,30 @@ <h1 id="search" class="anchor"><a href="#search"><i class="fas fa-link anchor-ic
 include cardv2.html
 title="DuckDuckGo"
 image="/assets/img/svg/3rd-party/duckduckgo.svg"
-description='DuckDuckGo is a "search engine that doesn\'t track you." Some of DuckDuckGo\'s code is free software hosted at GitHub, but the core is proprietary. <span class="flag-icon flag-icon-us"></span> <a href="../../providers/#ukusa">The company is based in the USA.</a>'
+description='DuckDuckGo is a "search engine that doesn\'t track you." Some of DuckDuckGo\'s code is free software hosted at GitHub, but the core is proprietary. <span class="flag-icon flag-icon-us"></span> <a href="../../providers/#ukusa"> The company is based in the USA.</a>'
```
Comment (dawidpotocki) [inline _includes/sections/search-engines.html:22]: 🤔

File: _includes/sections/search-engines.html [hunk 3]
```diff
@@ -19,19 +19,30 @@ <h1 id="search" class="anchor"><a href="#search"><i class="fas fa-link anchor-ic
...
+privacy-policy="https://www.startpage.com/en/privacy-policy/"
+forum="https://forum.privacytools.io/t/delisted-discussion-startpage/284"
+%}
+
 {%
 include cardv2.html
 title="Qwant"
 image="/assets/img/svg/3rd-party/qwant.svg"
-description='Qwant is a search engine with its philosophy based on two principles: no user tracking and no filter bubble. <span class="flag-icon flag-icon-fr"></span> <a href="../../providers/#ukusa">The company is based in France.</a>'
+description='Qwant is a search engine with its philosophy based on two principles: no user tracking and no filter bubble. <span class="flag-icon flag-icon-fr"></span> <a href="../../providers/#ukusa"> The company is based in France.</a>'
```
Comment (dawidpotocki) [inline _includes/sections/search-engines.html:45]: 🤔

**Pull request merged**